l1-contracts/src/core/libraries/validator-selection/ValidatorSelectionLib.sol

```diff
@@ -227,12 +230,13 @@ library ValidatorSelectionLib {
   }
 
   /**
-   * @notice  Sets the sample seed for the next epoch
+   * @notice  Sets the sample seed for the epoch after the next
    *
    * @param _epoch - The epoch to set the sample seed for
    */
   function setSampleSeedForNextEpoch(Epoch _epoch) internal {
-    setSampleSeedForEpoch(_epoch + Epoch.wrap(1));
+    // Q: If the seed for the immediately next epoch was not set, should we set it?
```

Comment:
if we do we end up in the state where we can set it at an epoch boundary and it will be able to change